### PR TITLE
1 warning, 1 info on tasks

### PR DESCRIPTION
**Pull request by why-not-try-calmer**
Goal:
- clarify a few things on tasks

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

File: docs/pyqgis_developer_cookbook/tasks.rst [hunk 1]
```diff
@@ -99,6 +99,17 @@ There are several ways to create a QGIS task:
    Attempting to use them from background threads will result in
    crashes.
 
+   Moreover always make sure that `context` and `feedback` live for at
+   least as long as any `QgsTask` using them. Qgis will crash should a task
```
Comment (Gustry) [inline docs/pyqgis_developer_cookbook/tasks.rst:103]:
```suggestion
   least as long as any `QgsTask` are using them. QGIS will crash should a task
```

Comment (Gustry) [reply docs/pyqgis_developer_cookbook/tasks.rst:103]: Can you review the sentence "QGIS will crash should a task... ? Look strange

**Review by Gustry — COMMENTED** [review]
I'm not a native speaker, just reviewing quickly ;-)

File: docs/pyqgis_developer_cookbook/tasks.rst (hunk 1 above)
Comment (why-not-try-calmer) [reply docs/pyqgis_developer_cookbook/tasks.rst]: Should be easier to parse now.

**Review by DelazJ — COMMENTED** [review]
@why-not-try-calmer Thanks. From a HTML rendering pov, using single bactick (\`) does make no change; the text will render with normal formatting. If you want text shown in red, you should double it (\`\`); Either we remove it.

**Comment by why-not-try-calmer** [comment]
I am trying to refer to the QgsProcessingContext's constructure. I have tried:
> :meth:`QgsProcessingContext() <qgis.core.QgsProcessingContext>`
> :meth:`QgsProcessingContext() <qgis.core.QgsProcessingContext.QgsProcessingContext>`

Any idea?

**Pull request merged**